ui/package.json

```diff
@@ -7,7 +7,7 @@
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build --configuration production --output-path=../public --source-map=false --aot=true",
-    "lint": "ng lint"
+    "lint": "ng lint --fix"
```

Comment:
not sure about automated fixes, after the await issues

Reply:
did not mean to keep this there, I only used it to auto sort all the imports/exports, forgot to remove it again!